courses-fe/src/pages/landingPage/LandingPageContainer.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from "react";
+import LandingPage from "./LandingPageComponent";
+
+class LandingPageContainer extends Component {
+
+  render() {
+    return <LandingPage {...this.props}/>;
```

Comment:
La fel si aici...vrei sa faci render la LandingPageComponent....doar in index o denumesti simplu LandingPage ... aici respectam arhitectura de Component si Container ca sa fie mai usor de urmarit